instaloader/instaloader.py

```diff
@@ -1403,6 +1428,7 @@ def check_profile_id(self, profile_name: str, latest_stamps: Optional[LatestStam
     def download_profiles(self, profiles: Set[Profile],
                           profile_pic: bool = True, posts: bool = True,
                           tagged: bool = False,
+                          reels: bool = False,
```

Comment:
Unfortunately even though this may seem a good place for the reels parameter it is not backwards-compatible fpr callers that pass the arguments positionally rather than by name. So we need to add the new parameter at the end of the list.

Reply:
Good catch.